docs/dir_49e56c817e5e54854c35e136979f97ca.html

```diff
@@ -110,16 +118,36 @@
 </div>
 
 <div class="header">
+<<<<<<<< HEAD:docs/structsirfile.html
+  <div class="headertitle"><div class="title">sirfile Struct Reference</div></div>
+</div><!--header-->
+<div class="contents">
+
+<p>Log file data.  
+ <a href="structsirfile.html#details">More...</a></p>
+
+<p><code>#include &lt;<a class="el" href="sirtypes_8h_source.html">sirtypes.h</a>&gt;</code></p>
+<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
+<div class="textblock"><p>Log file data. </p>
+</div><hr/>The documentation for this struct was generated from the following file:<ul>
+<li><a class="el" href="sirtypes_8h_source.html">sirtypes.h</a></li>
+</ul>
+========
   <div class="headertitle"><div class="title">docs Directory Reference</div></div>
 </div><!--header-->
 <div class="contents">
+>>>>>>>> master:docs/dir_49e56c817e5e54854c35e136979f97ca.html
 </div><!-- contents -->
 </div><!-- doc-content -->
 <!-- HTML footer for doxygen 1.9.7-->
 <!-- start footer part -->
 <div id="nav-path" class="navpath"><!-- id is needed for treeview function! -->
```

Comment:
I need to fix the Doxygen config; I don't want any of these internal structures to end up in the public documentation. There's no point to it–anyone who wants to hack on the library is going to have to learn much more than I can type in a little blurb, and there   are too many blurbs.